src/main/java/net/atos/entng/calendar/services/impl/EventServiceMongoImpl.java

```diff
@@ -202,7 +206,12 @@ public void create(String calendarId, JsonObject body, UserInfos user, Handler<E
         if (body.getValue("calendar") == null) {
             body.put("calendar", new JsonArray().add(calendarId));
         }
-        mongo.save(this.collection, body, validActionResultHandler(handler));
+
+        if(collection != null) {
+            mongo.save(collection, body, validActionResultHandler(handler));
```

Comment:
` mongo.save(collection !== null) : collection : this.collection , body, validActionResultHandler(handler));`